docs/CHANGELOG.asciidoc

```diff
@@ -780,6 +773,17 @@ APM::
 * Adds pagination to source map API {kibana-pull}145959[#145959]
 * Adds ability to offset point labels on maps {kibana-pull}145773[#145773]
 
+Cases::
+* Adds new column `Updated on ` in `all cases list ` table. This column can be sorted and can persist sorting options {kibana-pull}149116[#149116]
+* Redesign all cases list select modal {kibana-pull}149851[#149851]
+* RenderCustomActionsRow with named params instead of args {kibana-pull}149304[#149304]
```

Comment:
Thanks for the changes! This should not be part of the Cases section.

Reply:
Oops! Good catch - I moved 149304 back to alerting.

Addressed in [a5d7c8d](https://github.com/elastic/kibana/pull/157805/commits/a5d7c8dd176cb1e880ae7b6e318b68a5084c3347).